zilliqa/src/contracts/deposit.sol

```diff
@@ -209,6 +218,17 @@ contract Deposit {
         return _stakerKeys;
     }
 
+    function getStakerSignerAddresses() public view returns (address[] memory) {
```

Comment:
Efficiency basically, we want to retrieve & set the validator addresses on the validator manager contract in one go.

Reply:
Not really, because that will require code changes for no good reason. What's the problem with adding and using this?

Reply:
Because I'd like to maintain a consistent approach within `main` at all times. If we have multiple ways of accessing the same data, then a developer reading the deposit contract won't know what the correct way is and a developer adding new data won't know what accessors to add.

Sorry for being nitpicky here, but the deposit contract is an important part of the system and a public interface that we want to keep clean.

Reply:
Talk to @DrZoltanFazekas about this. We intentionally added this after a discussion. I don't want to waste time for a redundant change that will be implemented like this later anyway.